exotations/task_maps/task_map/exotica_plugins.xml

```diff
@@ -44,4 +44,13 @@
   <class name="exotica/QuasiStatic" type="exotica::QuasiStatic" base_class_type="exotica::TaskMap">
     <description>Plannar quasi static cost/constraint</description>
   </class>
+  <class name="exotica/JointVelBackFiniteDiff" type="exotica::JointVelBackFiniteDiff" base_class_type="exotica::TaskMap">
+    <description>Smooths joint velocities.</description>
```

Comment:
A little more information is required: "Smooths joint velocities by penalising ... through backward finite differencing." <-- this makes it clear it is not the joint velocity smoothing for trajectories, but only for end poses. Which should also be added as a limitation.

``Taskmap for use with single-step/EndPose problems only``.

Reply:
Please add as in prior comment suggested it’s applied at every single timeatep individually und for endposes and not trajectories